pkg/util/azureclient/azuresdk/armstorage/blobs.go

```diff
@@ -0,0 +1,27 @@
+package armstorage
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+
+	"github.com/openshift/installer-aro-wrapper/pkg/env"
+)
+
+type BlobStorageClient interface {
```

Comment:
I think this isn't needed or used.  No point in defining an empty interface.

Reply:
deleted the whole file, it is not used and needed.